src/Arcus.Messaging.Abstractions/Telemetry/MessageCorrelationInfoEnricher.cs

```diff
@@ -8,6 +8,7 @@ namespace Arcus.Messaging.Abstractions.Telemetry
     /// <summary>
     /// Logger enrichment of the <see cref="MessageCorrelationInfo" /> model.
     /// </summary>
+    [Obsolete("Will be moved in v3.0 outside the 'Abstractions' library in a separate Serilog-specific library, see the v3.0 migration guide for more information")]
```

Comment:
I'd only do this once the alternative exists.

Reply:
Yes, that's what the message says. Think this is also not often used directly.